CHI_TECH/ChiMesh/Cell/cell_slab.h

```diff
@@ -15,6 +15,15 @@ class chi_mesh::CellSlab : public chi_mesh::Cell
   int     v_indices[2];
   int     edges[2];
   Vector  face_normals[2];
+
+public:
+  CellSlab() : Cell(CellTypes::SLAB_CELL)
```

Comment:
I don't mind revisiting again – but for future reference:  initializing in the list  would be better. Also (**important**), note that since `face_normals` is a traditional C array of `Vector`s, it's elements will also be left uninitialized unless you do it explicitly. 

This would be better here:

```c++
CellSlab() : Cell(CellType::SLAB), v_indices{-1,1}, edges{0,1} 
{ }
```

I

Reply:
Oh sheeeep! Yes. I literally brain farted this one.